File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
      l = 0;
      length = len(nums)
      for r in nums:
        if nums[l] != r:
          l+=1
          nums[l] = r
        print(l)
      return l + 1

File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.ts
function removeDuplicates(nums: number[]): number {
  
  let r = 1
  for(let i = 0; i < nums.length - 1; i++) {
    if(nums[i] !== nums[i + 1]) {
      nums[r] = nums[i + 1]
      r++
    }
  }
  return r
};